packages/replay-next/components/Icon.tsx

```diff
@@ -627,6 +633,10 @@ export default function Icon({
       path =
         "M4 12a8 8 0 018-8V0C5.373 0 0 5.373 0 12h4zm2 5.291A7.962 7.962 0 014 12H0c0 3.042 1.135 5.824 3 7.938l3-2.647z";
       break;
+    case "submit-ticket":
+      path =
+        "M6.02885 9.59588C5.8592 9.59588 5.71785 9.53434 5.60475 9.4112C5.49732 9.28192 5.4436 9.12495 5.4436 8.94027C5.4436 8.76174 5.49732 8.61091 5.60475 8.48778C5.71785 8.36467 5.8592 8.3031 6.02885 8.3031H11.4996C11.6636 8.3031 11.8021 8.36467 11.9152 8.48778C12.0339 8.61091 12.0933 8.76174 12.0933 8.94027C12.0933 9.12495 12.0339 9.28192 11.9152 9.4112C11.8021 9.53434 11.6636 9.59588 11.4996 9.59588H6.02885ZM6.02885 12.0614C5.8592 12.0614 5.71785 11.9998 5.60475 11.8767C5.49732 11.7475 5.4436 11.5935 5.4436 11.415C5.4436 11.2365 5.49732 11.0857 5.60475 10.9626C5.71785 10.8333 5.8592 10.7686 6.02885 10.7686H10.0322C10.1962 10.7686 10.3347 10.8333 10.4478 10.9626C10.5666 11.0857 10.626 11.2365 10.626 11.415C10.626 11.5935 10.5666 11.7475 10.4478 11.8767C10.3347 11.9998 10.1962 12.0614 10.0322 12.0614H6.02885ZM16.5293 12.3107C16.1561 12.3107 15.814 12.2122 15.503 12.0152C15.1977 11.8121 14.9517 11.5443 14.7651 11.2119C14.5841 10.8733 14.4937 10.5008 14.4937 10.0945C14.4937 9.68822 14.5841 9.31886 14.7651 8.98644C14.9517 8.64784 15.1977 8.38005 15.503 8.18305C15.814 7.9799 16.1561 7.87834 16.5293 7.87834C16.9025 7.87834 17.2418 7.9799 17.5471 8.18305C17.8524 8.38005 18.0956 8.64784 18.2765 8.98644C18.4631 9.31886 18.5565 9.68822 18.5565 10.0945C18.5565 10.5008 18.4631 10.8733 18.2765 11.2119C18.0956 11.5443 17.8524 11.8121 17.5471 12.0152C17.2418 12.2122 16.9025 12.3107 16.5293 12.3107ZM4.66327 21C3.77552 21 3.10829 20.76 2.66158 20.2798C2.22053 19.8058 2 19.0947 2 18.1467V6.86258C2 5.90839 2.22053 5.19429 2.66158 4.72026C3.10829 4.24009 3.77552 4 4.66327 4H19.3367C20.2302 4 20.8973 4.24009 21.3385 4.72026C21.7795 5.20044 22 5.91454 22 6.86258V18.1467C22 19.0947 21.7795 19.8058 21.3385 20.2798C20.8973 20.76 20.2302 21 19.3367 21H4.66327ZM4.68025 19.5134H19.3198C19.7382 19.5134 20.0605 19.3933 20.2867 19.1532C20.5185 18.9069 20.6343 18.5438 20.6343 18.0636V6.94569C20.6343 6.46551 20.5185 6.10231 20.2867 5.85606C20.0605 5.60981 19.7382 5.4867 19.3198 5.4867H4.68025C4.25615 5.4867 3.93102 5.60981 3.70483 5.85606C3.47866 6.10231 3.36556 6.46551 3.36556 6.94569V18.0636C3.36556 18.5438 3.47866 18.9069 3.70483 19.1532C3.93102 19.3933 4.25615 19.5134 4.68025 19.5134Z";
+      break;
```

Comment:
Technically these icons didn't need to move into this component, but I thought we'd be sharing them between multiple places (before it became clear that Jon wanted these forms to stay nested). This way they'll be easier to reuse if we want to in the future though.